packages/web/modals/validator-squad.tsx

```diff
@@ -21,37 +21,34 @@ import { ExternalLinkIcon, Icon } from "~/components/assets";
 import { Button } from "~/components/buttons";
 import { CheckBox } from "~/components/control";
 import { SearchBox } from "~/components/input";
+import { Tooltip } from "~/components/tooltip";
 import { useFilteredData } from "~/hooks";
 import { ModalBase, ModalBaseProps } from "~/modals/base";
 import { useStore } from "~/stores";
+import { theme } from "~/tailwind.config";
 import { normalizeUrl, truncateString } from "~/utils/string";
 
-interface ExtendedModalBaseProps extends ModalBaseProps {
+interface ValidatorSquadModalProps extends ModalBaseProps {
   usersValidatorsMap: Map<string, Staking.Delegation>;
   validators: Staking.Validator[];
 }
```

Comment:
Nit: It's nice how this extends `ModalBaseProps` as an interface, but conventionally we inline the types in the component signature. 

It's nice to have the types of the props right next to the props so we don't have to scroll up and down constantly.